### DISCUSSION TITLE
[WASM] project build failing on MacOS VsCode

### DISCUSSION THREAD

**Discussion by EunjeongChoi414** [unoplatform/uno#16891] [Q&A] [answered]
I keep getting this error message. 

/Users/jungchoi/.nuget/packages/uno.wasm.bootstrap/8.0.13/build/Uno.Wasm.Bootstrap.targets(213,3): error : System.Exception: Failed to run emscripten. 

I installed emscripten (emcc --version works) as well as python3, node.js, and java for it. I configured environment variables and tried deleting username/.uno/emsdk

I created a binlog but I couldn't attach binlog files here so I uploaded on my [repo](https://github.com/EunjeongChoi414/Binlog) instead. 

Is there anyone who can help me fix this issue 🤧

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report! At this time, building using [AOT is not supported on macOS](https://platform.uno/docs/articles/external/uno.wasm.bootstrap/doc/runtime-execution-modes.html#mixed-interpreter-and-aot-mode). You can use a linux container, or github actions to do so.

**Comment by EunjeongChoi414** [comment]
@jeromelaban Thank you for your quick response. Sadly Im still getting the same error even when I changed the execution mode to Interpreter as stated in the docs. I updated the [binlog](https://github.com/EunjeongChoi414/Binlog).

**Reply by jeromelaban** [reply]
Thank you for the update. There is an issue when using SkiaSharp on macOS, a dependency that is activated when using the recommended project template. We'll provide a fix for this (until net9 fixes macOS support), but in the meantime, you can use the "blank" uno platform template instead.
